textattack/transformations/word_swaps/word_swap_change_name.py

```diff
@@ -40,13 +41,20 @@ def __init__(
         self.first_only = first_only
         self.last_only = last_only
         self.confidence_score = confidence_score
+        self.language = language
 
     def _get_transformations(self, current_text, indices_to_modify):
         transformed_texts = []
+        if self.language == "en":
+            model_name = "ner"
+        elif self.language == "fra" or self.language == "french":
+            model_name = "flair/ner-french"
+        else:
```

Comment:
so here we do not need something like follows? :

`  if self.language == "esp" or self.language == "spanish":`

Reply:
We do not need to do self.language == "esp" or self.language == "spanish" because the model we use for spanish is a multilingual model from flair(https://huggingface.co/flair/ner-multi-fast) includes german dutch spanish and english. It however only "kind of works" for related languages such as french, so french is the only language that needs a separated model. We so far do not have support for german and dutch yet, but if we were to add them they should work without changing the logic here.

Reply:
I see. this works!